packages/client/hmi-client/src/stores/opened-workflow-node.ts

```diff
@@ -1,16 +1,28 @@
 import { defineStore } from 'pinia';
 import { ProjectAssetTypes } from '@/types/Project';
 
+interface StringValueMap {
+	[key: string]: number;
+}
+
 // Probably will be an array later
 export const useOpenedWorkflowNodeStore = defineStore('opened-workflow-node', {
 	state: () => ({
 		assetId: null as string | null,
-		pageType: null as ProjectAssetTypes | null
+		pageType: null as ProjectAssetTypes | null,
+		openedOutputIndex: 0 as number,
+		// model node
+		initialValues: null as StringValueMap[] | null,
```

Comment:
Nitpick; maybe we name these `modelConfigInitialValues` and `modelConfigParameterValues` so that we know it's explicitly related to model node functionality? Or maybe wrap the two fields in a `modelConfig` object?